scripts/drop_sweep_entry.py

```
@@ -0,0 +1,105 @@
+            and (args.config is None or r["config"] == args.config)
+            and (args.seed is None or str(r["seed"]) == str(args.seed))
+        )
+
+    sel = f"name={args.name}"
+    if args.config is not None:
+        sel += f", config={args.config}"
+    if args.seed is not None:
+        sel += f", seed={args.seed}"
+
+    if args.dry_run:
+        for path, fields in ((args.output, sc.FIELDNAMES),
```

Ruff (0.15.18)

[warning] Loop control variable `fields` not used within loop body

Rename unused `fields` to `_fields`

(B007)